CHANGELOG.md

```diff
@@ -13,8 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Updated documentation for MeetingSessionStatusCode to clarify the origin and meaning of each. Included information on use of isTerminal flag to determine if retry will be automatically attempted.
```

Comment:
1. use inline code block (backtick key) to highlight the term.`MeetingSessionStatusCode`, `isTerminal`
2. use present tense "Updated" -> "Update"